src/Controllers/Admin/GeneralSettingsController.php

```diff
@@ -27,5 +27,27 @@ public function update( Request $request, Response $response ) {
 		// Add the settings update code here
 
 		return new RedirectResponse( 302, admin_url( 'admin.php?page=dt_launcher&tab=general&updated=true' ) );
+		exit();
+
+	}
+
+	public function update_user_access_settings() {
```

Comment:
@Perumal605, please return a Redirect from the method instead of using wp_redirect